### PR TITLE
DOCSP-30898: v4.10 release

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-30898
Staging:
[What's New](https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-30898-v4.10-updates/whats-new/#std-label-version-4.10)
Compat: [mongodb](https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-30898-v4.10-updates/compatibility/#mongodb-compatibility) / [lang](https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-30898-v4.10-updates/compatibility/#language-compatibility)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: .github/.backportrc.json
Comment (mongoKart) [inline .github/.backportrc.json:1]: Do you know the reason for having two backportc files?

File: source/whats-new.txt [hunk 1]
```diff
@@ -42,12 +42,30 @@ Upcoming Breaking Changes
 What's New in 4.10
 ------------------
 
+.. important:: Aggregation Expression Operations are in Beta
```
Comment (mongoKart) [inline source/whats-new.txt:45]:
```suggestion
.. important:: Aggregation Expression Operations Are in Beta
```

File: source/whats-new.txt [hunk 2]
```diff
@@ -42,12 +42,30 @@ Upcoming Breaking Changes
...
+   
+   Version 4.9 of the {+driver-short+} introduced an API for creating
+   :ref:`aggregation expressions <java-aggregation-expression-operations>`. 
+   This feature is still in beta phase.
```
Comment (mongoKart) [inline source/whats-new.txt:49]: Did anything change between 4.9 and 4.10? Wondering if we need to call out that it's still in beta.

**Review by mongoKart — CHANGES_REQUESTED** [review]
Great job. One change + two questions

File: .github/.backportrc.json
Comment (rustagir) [reply .github/.backportrc.json:1]: deleting the one from `.github`

File: source/whats-new.txt (hunk 2 above)
Comment (rustagir) [reply source/whats-new.txt:49]: I'm going to move this note to v4.9

**Review by mongoKart — APPROVED** [review]
lgtm!

**Pull request merged**